src/pages/About.tsx

```diff
@@ -1,8 +1,71 @@
 import { FC } from 'react';
-const About: FC = () => (
-  <div>
-    <h1>About</h1>
-  </div>
-);
+import Page from '../ui/Page';
+import styled from '@emotion/styled';
+import Button from '../ui/form/Button';
+import { useNavigate } from 'react-router-dom';
+import Flex from '../ui/Flex';
+
+interface BarProps {
+  left: string;
+  top: string;
+  rotate: string;
+  background: 'green' | 'salmon' | 'red' | 'blue' | 'paper' | 'purple' | 'ochre' | 'pink';
```

Comment:
nitpicky, this can just be string, then pass the string 'var(--red)' but this is fine too if a little bit strict